src/BuiltInTools/dotnet-watch/Filters/BrowserRefreshFilter.cs

```diff
@@ -10,6 +10,8 @@ namespace Microsoft.DotNet.Watcher.Tools
 {
     internal sealed class BrowserRefreshFilter : IWatchFilter, IAsyncDisposable
     {
+        // This needs to be in sync with the version BrowserRefreshMiddleware is compiled against.
```

Comment:
Should we add a comment about this here?

https://github.com/dotnet/sdk/blob/45e21d9c56cc7d3c8b6cb2db53d0c63dfe47e1bb/src/BuiltInTools/BrowserRefresh/Microsoft.AspNetCore.Watch.BrowserRefresh.csproj#L3-L4

I figure that's where people changing the BrowserRefresh TFM would likely see the comment.